src/main/java/uk/gov/ons/response/survey/representation/SurveyDTO.java

```diff
@@ -11,10 +11,17 @@
 @Data
 @NoArgsConstructor(access = AccessLevel.PUBLIC)
 public class SurveyDTO {
+
+  public enum SurveyType {
```

Comment:
Do I miss something ?Is this enum used to store enumerated values in the db ? If so SurveyType should not be  @Enumerated(EnumType.STRING) and why is it public ?

Reply:
It comes from a REST API and I guess the enum is public so it can be referenced i.e. SurveyType.valueOf("Business")